docs/src/modules/branding/BrandingSupportTheProject.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import Container from '@material-ui/core/Container';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import Button from '@material-ui/core/Button';
+import QuestionAnswerIcon from '@material-ui/icons/QuestionAnswer';
+import UnderlinedText from 'docs/src/modules/branding/UnderlinedText';
+import FeedbackIcon from 'docs/src/modules/branding/icons/Feedback';
+import ChangesIcon from 'docs/src/modules/branding/icons/Changes';
+import FinanceIcon from 'docs/src/modules/branding/icons/Finance';
+import HelpIcon from 'docs/src/modules/branding/icons/Help';
+import OpenCollectiveIcon from 'docs/src/modules/branding/icons/OpenCollective';
+
+interface SupportCardProps {
+  color?: 'primary' | 'info';
+  icon?: React.ReactNode;
+  children?: React.ReactNode;
+  title?: string;
+}
+
+const SupportCard = styled((props: SupportCardProps) => {
+  const { color, icon, title, children, ...other } = props;
+  return (
+    <Card elevation={0} {...other}>
+      <CardHeader avatar={<Avatar aria-label={title}>{icon}</Avatar>} />
+      <CardContent>
+        <Typography variant="h3">{title}</Typography>
+        <Typography component="div">{children}</Typography>
+      </CardContent>
+    </Card>
+  );
+})<SupportCardProps>(({ color = 'primary', theme }) => ({
+  '& [class*="MuiAvatar-root"]': {
+    background: color === 'info' ? theme.palette.vividBlue : theme.palette.primary.main,
+    width: 80,
+    height: 80,
+  },
+}));
+
+export default function BrandingSupportTheProject() {
+  return (
+    <Container sx={{ mt: [10, 18], mb: [12, 20] }}>
+      <Typography variant="h2">Material-UI is awesome.</Typography>
+      <Typography variant="h2">
```

Comment:
I'm not sure it's usual to find two h2 side by side in the DOM. I have no idea how it's handled by screenreaders and SEO engines. Have you considered to have a single `<h2>` with a `<be />`?

Reply:
Makes sense, will change it